website/source/docs/agent/config-entries/service-defaults.html.md

```diff
@@ -41,6 +41,11 @@ Protocol = "http"
 
   - `Mode` `(string: "")` - One of `none`, `local`, or `remote`.
 
+- `ExternalSNI` <sup>(beta)</sup> `(string: "")` - This is an optional setting
```

Comment:
While I was here and had to link to it, I slipped in this missing documentation.